### PR TITLE
chore: Apply fixes that clippy insists on

### PR DESCRIPTION
**Pull request by roypat**
I keep accidentally adding these to random commit because of my pre-commit hook, so here they are.

## Changes

...

## Reason

...

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license. For more information on following
Developer Certificate of Origin and signing off your commits, please check
[`CONTRIBUTING.md`][3].

## PR Checklist

- [ ] If a specific issue led to this PR, this PR closes the issue.
- [ ] The description of changes is clear and encompassing.
- [ ] Any required documentation changes (code and docs) are included in this PR.
- [ ] API changes follow the [Runbook for Firecracker API changes][2].
- [ ] User-facing changes are mentioned in `CHANGELOG.md`.
- [ ] All added/changed functionality is tested.
- [ ] New `TODO`s link to an issue.
- [ ] Commits meet [contribution quality standards](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md#contribution-quality-standards).

---

- [ ] This functionality cannot be added in [`rust-vmm`][1].

[1]: https://github.com/rust-vmm
[2]: https://github.com/firecracker-microvm/firecracker/blob/main/docs/api-change-runbook.md
[3]: https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md

### CODE REVIEW

File: src/vmm/tests/io_uring.rs [hunk 1]
```diff
@@ -120,7 +120,7 @@ fn test_eventfd() {
     let user_data: u8 = 71;
     let buf = [0; 4];
     let epoll = Epoll::new().unwrap();
-    let mut ready_events = vec![EpollEvent::default(); 1];
+    let mut ready_events = [EpollEvent::default(); 1];
```
Comment (ShadowCurse) [inline src/vmm/tests/io_uring.rs:124]: nit: maybe reword the commit message to be less annoyance driven?

File: src/vmm/tests/io_uring.rs [hunk 2]
```diff
@@ -120,7 +120,7 @@ fn test_eventfd() {
...
```
Comment (JonathanWoollett-Light) [inline src/vmm/tests/io_uring.rs:123]:
```suggestion
let mut ready_events = [EpollEvent::default()];
```

Comment (roypat) [reply src/vmm/tests/io_uring.rs]: Done!

File: src/vmm/tests/io_uring.rs (hunk 1 above)
Comment (roypat) [reply src/vmm/tests/io_uring.rs:124]: Done :)

**Review by ShadowCurse — APPROVED** [review]
